oauth-common/src/main/java/io/strimzi/kafka/oauth/common/Config.java

```diff
@@ -25,11 +29,13 @@ public class Config {
     public static final String OAUTH_ACCESS_TOKEN_IS_JWT = "oauth.access.token.is.jwt";
     public static final String OAUTH_CONNECT_TIMEOUT_SECONDS = "oauth.connect.timeout.seconds";
     public static final String OAUTH_READ_TIMEOUT_SECONDS = "oauth.read.timeout.seconds";
+    public static final String OAUTH_CONFIG_ID = "oauth.config.id";
+    public static final String OAUTH_ENABLE_METRICS = "oauth.enable.metrics";
```

Comment:
I don't know, we already have several boolean config options like: `oauth.check.issuer`, `oauth.check.access.token.type`, `oauth.check.audience`, `oauth.fail.fast`, `strimzi.authorizer.grant.when.no.delegate`, `strimzi.authorization.delegate.to.kafka.acl` ...

The common thread is that we name them using `imperative` mode rather than `passive`.
`oauth.enable.metrics` continues that theme, whereas `oauth.metrics.enable` breaks it in my opinion.